doc/classes/Path2D.xml

```diff
@@ -9,9 +9,55 @@
 	</description>
 	<tutorials>
 	</tutorials>
+	<methods>
+		<method name="is_debug_global_enabled" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if the global path debug is enabled.
+			</description>
+		</method>
+		<method name="set_debug_global_enabled">
+			<return type="void" />
+			<param index="0" name="enabled" type="bool" />
+			<description>
+				If [code]true[/code], enables the global path debug.
+			</description>
+		</method>
+	</methods>
 	<members>
 		<member name="curve" type="Curve2D" setter="set_curve" getter="get_curve">
 			A [Curve2D] describing the path.
 		</member>
+		<member name="debug_custom_color" type="Color" setter="set_debug_custom_color" getter="get_debug_custom_color" default="Color(1, 1, 1, 1)">
+			The custom color used to draw the path debug when [member debug_custom_enabled] is enabled.
+		</member>
+		<member name="debug_custom_enabled" type="bool" setter="set_debug_custom_enabled" getter="get_debug_custom_enabled" default="false">
+			If [code]true[/code], the debug custom properties override the global defaults.
+		</member>
```

Comment:
Note that a similar API in **CollisionShape2D** doesn't work like this. If `CollisionShape2D.debug_color` is `Color(0, 0, 0, 0)`, the value defined in the project settings is used. One could make similar arguments for the other properties, but this is the most striking.

Reply:
Using any magic values to control debug is not a great because not only do you lay a trap for people that want to use that value but you force people to script book-keep that value when they want to toggle between custom and global debug.

Reply:
In the case of colour it seems completely fine, and Godot is no stranger to it. Some color theme properties behave the same.

Either way, what's important to me here is that a different implementation for a similar concept already exists. Whether you disagree with it (I sympathise), this inconsistency will be a bit of a head scratcher. **CollisionShape3D** also has `debug_color`.